schema/GenomeAnnotationEnum.mcf

```diff
@@ -997,3 +997,162 @@ name: "Estonian Genome Project"
 url: "https://genomics.ut.ee/en/about-us/estonian-genome-centre/"
 typeOf: dcs:GenVarSourceEnum
 
+Node: dcid:GenomeAssemblyTypeEnum
+name: "GenomeAssemblyTypeEnum"
+typeOf: schema:Class
+subClassOf: schema:Enumeration
+description: "The type of sequence model data defined in this genome assembly."
+
+Node: dcid:GenomeAssemblyTypeAssembly
+name: "GenomeAssemblyTypeAssembly"
+typeOf: dcs:"GenomeAssemblyTypeEnum"
+descriptionUrl: "https://www.ncbi.nlm.nih.gov/assembly/model/#asmb_def"
+description: "The set of chromosomes, unlocalized and unplaced (sometimes called 'random') and alternate sequences used to represent an organism's genome. Assemblies are constructed from 1 or more assembly units."
+
+Node: dcid:GenomeAssemblyTypeHaploidAssembly
```

Comment:
Just curious, is GenomeAssemblyTypeHaploidAssembly a sub-category of GenomeAssemblyTypeAssembly? If so we can use the specializationOf property to link to GenomeAssemblyTypeAssembly.

Reply:
+1

Reply:
Added specializationOf property to the subcategories of that enumeration